mirage/scenarios/default.ts

```diff
@@ -42,6 +45,8 @@ export default function(server: Server) {
         index: 0,
     });
 
+    const firstNode = server.create('node', { affiliatedInstitutions });
```

Comment:
Not actually the first node. Do we need this given the one we create below? Maybe we should randomly give nodes an affiliated institution? Would probably be good to use a trait for this.